src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -52,9 +54,12 @@ export default function withAnnotations<P extends object>(
         }
 
         handleAnnotationCreate = (eventData: AnnotationActionEvent): void => {
-            const { annotation = null, error = null } = eventData;
+            const { annotation = null, error = null, meta = null } = eventData;
```

Comment:
The default value of `meta` can be `{}` then you don't have to do `const { requestId } = meta || {};` in the next file.